src/components/ViewRequest/Documents/DocumentView.tsx

```diff
@@ -0,0 +1,118 @@
+import { Icon } from "@fluentui/react";
+import {
+  Button,
+  Caption1,
+  Card,
+  CardHeader,
+  Link,
+  Text,
+} from "@fluentui/react-components";
+import { getFileTypeIconProps } from "@fluentui/react-file-type-icons";
+import { DeleteIcon } from "@fluentui/react-icons-mdl2";
+import { SPDocument, useDeleteDocument } from "api/documentsApi";
+
+export const DocumentView = (props: {
+  document: SPDocument;
+  readonly?: boolean;
+}) => {
+  const deleteDocument = useDeleteDocument(props.document);
+
+  const extension = props.document.ServerRelativeUrl.substring(
+    props.document.ServerRelativeUrl.lastIndexOf(".") + 1
+  );
+
+  const isOfficeFile: boolean = wordExtensions
+    .concat(excelExtensions)
+    .concat(ppExtensions)
+    .includes(extension);
+
+  const viewEdit = props.readonly ? "v" : "e";
+
+  //Application specific URI schemes
+  let downloadUrl = props.document.ServerRelativeUrl;
+  if (wordExtensions.includes(extension)) {
+    downloadUrl = `ms-word:of${viewEdit}|u|${window.location.origin}${props.document.ServerRelativeUrl}`;
+  } else if (excelExtensions.includes(extension)) {
+    downloadUrl = `ms-excel:of${viewEdit}|u|${window.location.origin}${props.document.ServerRelativeUrl}`;
+  } else if (ppExtensions.includes(extension)) {
+    downloadUrl = `ms-powerpoint:of${viewEdit}|u|${window.location.origin}${props.document.ServerRelativeUrl}`;
+  }
+
+  return (
+    <Card style={{ margin: "0.5em" }}>
+      <CardHeader
+        image={
+          <Icon
+            {...getFileTypeIconProps({
+              extension: extension,
+              size: 64,
+              imageFileType: "png",
+            })}
+          />
+        }
+        header={
+          <Link download={!isOfficeFile} href={downloadUrl}>
+            <Text size={600} weight="semibold">
```

Comment:
Would a Title3 or Subtitle1 be a good preset to use?

Reply:
Fluent UI just converts it to a plain old span with classes applied.  I think it being a Title or Subtitle semantically makes sense with typography etc too, as it is the title of that document card.  I'm good either way, just thought I would bring it up as something to think about/consider.